joomla/joomla-cms
# [5.1] Fix Fontend language multilingual without compatibility plugin

### Summary of Changes
Fix wrong field name for multilingual sites in user profile


### Testing Instructions
- Disable the "Behaviour - Backward Compatibility" plugin
- Enable the "System - Language Filter" plugin
- Check User profile (frontend/backend)


### Actual result BEFORE applying this Pull Request
- Language field is a text field


### Expected result AFTER applying this Pull Request
- Language field is a select
